src/tests/system/tests/test_ipa.py

```diff
@@ -0,0 +1,91 @@
+"""
+SSSD IPA tests.
+
+:requirement: ipa
+"""
+
+from __future__ import annotations
+
+import time
+
+import pytest
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.roles.ipa import IPA
+from sssd_test_framework.topology import KnownTopology
+
+
+@pytest.mark.ticket(bz=1926622)
+@pytest.mark.importance("critical")
+@pytest.mark.topology(KnownTopology.IPA)
+def test_ipa_add_support_to_verify_authentication_indicators(client: Client, ipa: IPA):
```

Comment:
test_ipa__ (2 dashes) 

Let's find a better name, we are not adding support to verify, this is testing something, the title doesn't make sense.